packages/perseus/src/widgets/dropdown/dropdown.stories.tsx

```diff
@@ -43,6 +48,57 @@ export const DropdownWithEmptyPlaceholder = (
     return <RendererWithDebugUI question={dropdownWithEmptyPlaceholder} />;
 };
 
-export const DropdownWithNoAnswers = (args: StoryArgs): React.ReactElement => {
-    return <RendererWithDebugUI question={splitPerseusItem(basicDropdown)} />;
+export const AnswerlessBasicDropdown = (
+    args: StoryArgs,
+): React.ReactElement => {
+    return (
+        <ServerItemRendererWithDebugUI
+            item={basicDropdownItem}
```

Comment:
I think it's a little overkill to duplicate each story (really we just needed one interactive answerless story per widget). Having a bunch of stories that are nearly identical might be confusing to people especially in a couple of months when all stories should start answerless. Not too worried about this, but maybe we need a post-SSS ticket to come back and simplify the stories.

What I think is definitely overkill is exporting/importing what is essentially the same test data. Rather than redeclare the test data, just wrap the existing test data.

```suggestion
            item={createPerseusItem(basicDropdownItem)}
```

Reply:
Updated the stories to do inline item creation and reduced the overall number of stories, though I kept one for each type of interactive graph.